src/net.cpp

```diff
@@ -2407,10 +2408,14 @@ void CConnman::OpenNetworkConnection(const CAddress& addrConnect, bool fCountFai
     } else if (FindNode(std::string(pszDest)))
         return;
 
+    LogPrint(BCLog::NET_NETCONN, "CConnman::%s -- connecting to %s\n", __func__, addrConnect.ToString(false));
```

Comment:
These 3 new `LogPrint`s here should respect `fLogIPs`

Reply:
Fixed